lib/compareValues.js

```diff
@@ -78,7 +80,9 @@ function compareValues (a, b) {
     case 'NULL':
       return spaceship(a, b)
     case 'PTYPE<TIME>':
-      return spaceship(a.getTime(), b.getTime())
+      const timeA = _.isDate(a) ? a.getTime() : a.epoch_time * 1000
```

Comment:
Time as a primary key?

Does RethinkDB actually support this?

I can't imagine that would ever be a good idea.